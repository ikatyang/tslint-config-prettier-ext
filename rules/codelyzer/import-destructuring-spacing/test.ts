import {Foo,Bar} from './foo';
