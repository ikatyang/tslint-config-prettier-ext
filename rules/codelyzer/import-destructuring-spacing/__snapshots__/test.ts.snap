// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (import-destructuring-spacing) /rules/codelyzer/import-destructuring-spacing/test.ts[1, 8]: You need to leave whitespaces inside of the import statement's curly braces

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
import {Foo,Bar} from './foo';

======
import { Foo, Bar } from \\"./foo\\";

>>>>>> after
"
`;
